Gemfile.lock

```diff
@@ -52,9 +52,6 @@ GEM
     image_optim_pack (0.6.0.20200215)
       fspath (>= 2.1, < 4)
       image_optim (~> 0.19)
-    image_optim_pack (0.6.0.20200215-x86_64-darwin)
```

Comment:
Hi @jeff-matthews do you remember what I was trying to show you during the 1:1?, this is what happens, when I run rake to generate the local preview I get the gemfile.lock like it has been modified. So what I did was run bundle install and I got this into the PR. How can we fix it?, thanks!

Reply:
@danidelcar, I believe this happens when you run `bundle update` instead of `bundle install`. You should revert changes to this file.

Reply:
I'll check, thanks